controls/radvirtualgrid/localization.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Localization
+page_title: Localization
+description: Localization
+slug: virtualgrid-localization
+tags: localization
+published: True
+position: 10
+---
+
+# Localization
+
+> More detailed look over the localization capabilities that the framework provides can be found in the [Localization]({%slug common-localization%}) article.
+
+This topic will go through the approach of localizing __RadVirtualGrid__ component.
```

Comment:
I can't see any approach shown in the article that demonstrates how to localize RadVirtualGrid. May be you can:

- Modify this sentence to tell that it lists the supported language and available resources keys.
- Or show some custom localization. Or maybe show how to change the language of the application.